configuration/index.md

```diff
@@ -128,14 +130,37 @@ Every interface has it's own strong points.
   </tbody>
 </table>
 
-{: #text-vs-ui}
 ### Textual vs. Graphical Configuration
 
-With openHAB 1.x the smart home configuration was done via configuration files only.
-One of the most visible additions to openHAB 2 is the administrative web interface Paper UI.
+In openHAB 1.x, smart home configuration was done via configuration files only.
+openHAB 2 added the administrative web interface "Paper UI". It allows for configuration of the base system and setup of openHAB 2 add-ons.
+Please be aware that not every addon/binding of openHAB version 1.x has been updated to version 2 and thus to work with Paper UI.
+You can still run these but you need to use file configuration for them.
 
 Things and Items can either be defined and managed in configuration files or handled by Paper UI in a [system-side database]({{base}}/administration/jsondb.html).
-
-Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a `.items` configuration file.
-
-Configuration done in files will be synced to the database but configuration done in the database will not be written to files.
+Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a 
+`.items` configuration file.
+Configuration done in files will be synced to the database (so they are accessible in Paper UI) but configuration done in the database will not be written back to files, so if you restart openHAB, changes made in Paper UI won't survive and the files' configuration data will be used again.
+
+## Recommendation for New Users
+
+All interface options are supposed to work, still some of the more advanced settings might work better in one or the other
+setup. You have the full freedom of choice. On the other hand, this can be very confusing at times, particularly for new
+users. To help starters in narrowing down choices, here is some guidance that might also help to avoid some future pitfalls:
+
+* use Paper UI to setup and maintain the openHAB base system. Use it to define _which_ addons and bindings you want to install and to
```

Comment:
Please break down to one line per sentence, for better versioning.
(We should test this again for list items, but i think it works with multiple lines per list item.)

Please check the other list items also.